service/crt/service.go

```diff
@@ -23,11 +23,6 @@ import (
 )
 
 const (
-	CertificateListAPIEndpoint  string = "/apis/giantswarm.io/v1/certificates"
-	CertificateWatchAPIEndpoint string = "/apis/giantswarm.io/v1/watch/certificates"
-
-	TPRName        = "certificate"
-	TPRDomain      = "giantswarm.io"
 	TPRVersion     = "v1"
 	TPRDescription = "Managed certificates on Kubernetes clusters"
```

Comment:
This should be in the TPR repo. Lets track an issue for this in our boards in case we are not doing it right away.